Hymns/Tags/TagEntity.swift

```diff
@@ -69,13 +69,13 @@ extension TagColor {
         case .none:
             return Color.primary
         case .blue:
-            return Color(red: 2/255, green: 118/255, blue: 254/255, opacity: 1.0)
+            return CustomColors.foregroundBlue
         case .green:
-            return Color(red: 35/255, green: 190/255, blue: 155/255, opacity: 1.0)
+            return CustomColors.foregroundGreen
         case .yellow:
-            return Color(red: 176/255, green: 146/255, blue: 7/255, opacity: 1.0)
+            return CustomColors.foregroundYellow
         case .red:
-            return Color(red: 255/255, green: 0, blue: 31/255, opacity: 0.78)
+            return CustomColors.foregroundRed
```

Comment:
what's wrong with just putting the colors directly in here?

Reply:
It's just because I use them again in TagSelectedLabelView and there they are needed without TagColor being instantiated. I did do this in my recent commit. But it introduces several other problems as I mentioned above and I don't know if it is a commit worth keeping.